README.md

```diff
@@ -11,6 +11,14 @@ These add-ons extend the 'vanilla' OpenStack environment with value-added
 features that Rackspace has found useful, but are not core to deploying an
 OpenStack cloud.
 
+# Juno Support
+
+Juno support for MaaS is still accomplished within [os-ansible-deployment's
+Juno branch](http://git.openstack.org/cgit/stackforge/os-ansible-deployment/tree/?h=juno),
+which pulls in the [rpc-maas](https://github.com/rcbops/rpc-maas) repository.
+
+As of Kilo, this repository contains both the playbooks and checks code.
+
```

Comment:
"In Juno, the maas plugins code is contained in a separate repo, [rpc-maas](https://github.com/rcbops/rpc-maas), and the ansible code to deploy and configure the checks and alarms using these plugins is contained in tree in the [os-ansible-deployment](http://git.openstack.org/cgit/stackforge/os-ansible-deployment/tree/?h=juno) repo.

As of kilo, both the maas plugins, and the ansible code to deploy and configure the checks and alarms, are contained within this [rpc-openstack](https://github.com/rcbops/rpc-openstack) repo"

I think this reads a little more clearly